template-donation.php

```diff
@@ -0,0 +1,100 @@
+<?php
```

Comment:
Hey also confused where this came from? How do I test this page? Should this be a part of a different Pull Request?

Reply:
Haha, man...github is killing me. I updated my branch to the latest master...or so I thought. 

No idea how these got in there, especially a closed branch. 

Apparently I need some hand holding when I pull and merge, =(

Reply:
Hey I fixed up the merge stuff from Jen's old branch. I think the only thing left is the donation template.